cmd/cosign/cli/sign/sign_blob.go

```diff
@@ -67,19 +67,20 @@ func SignBlobCmd(ro *options.RootOptions, ko options.KeyOpts, regOpts options.Re
 
 	signedPayload := cosign.LocalSignedPayload{}
 
+	var rfc3161Timestamp *cbundle.RFC3161Timestamp
 	if ko.TSAServerURL != "" {
 		clientTSA, err := tsaclient.GetTimestampClient(ko.TSAServerURL)
 		if err != nil {
 			return nil, fmt.Errorf("failed to create TSA client: %w", err)
 		}
-		b64Sig := []byte(base64.StdEncoding.EncodeToString(sig))
 
-		respBytes, err := tsa.GetTimestampedSignature(b64Sig, clientTSA)
+		respBytes, err := tsa.GetTimestampedSignature(sig, clientTSA)
```

Comment:
Q: why are we using b64sig ? I remember we agreed on using base64 for the blob.

Reply:
We discussed this a bit in https://github.com/sigstore/timestamp-authority/issues/116, but the only reason base64 was used was because the interface exposed the signature as base64. I'd say it's better to use the raw bytes because you don't have any ambiguity (for example, some base64 encodings might allow variable numbers of `=`).

FWIW, every client can make their own decision on what's timestamped as long as the client's verification layer knows what was timestamped (like for example what you implemented for understanding attestations without signatures). But I think Cosign should stick with raw bytes (and this aligns nicely with the future bundle format that doesn't use base64).

cc @znewman01 and @asraa who I've chatted with about this